src/Fixer/Phpdoc/PhpdocAlignFixer.php

```diff
@@ -44,6 +45,7 @@
         'type',
         'var',
         'method',
+        'dataProvider',
```

Comment:
this one line and tests for it shall be raised against `2.2` branch

Reply:
Removed.